src/main/java/com/couchbase/lite/Database.java

```diff
@@ -87,7 +88,7 @@
     public static int DEFAULT_MAX_REVS = 20;
 
     private Store store = null;
-    private String path;
+    final private String path;
```

Comment:
I think in the other classes, we use `private final`. It's better to have one style.